README.md

```diff
@@ -6,12 +6,118 @@ output:
 ---
 
 <!-- badges: start -->
-  [![R-CMD-check](https://github.com/Aggregate-Genius/periscope2/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/Aggregate-Genius/periscope2/actions/workflows/R-CMD-check.yaml)
+[![CRAN status](https://www.r-pkg.org/badges/version/periscope2)](https://CRAN.R-project.org/package=periscope2)
+[![Downloads Total](http://cranlogs.r-pkg.org/badges/grand-total/periscope2)](https://cran.r-project.org/package=periscope2)
+[![CircleCI build status](https://circleci.com/gh/Aggregate-Genius/periscope2.svg?style=svg)](https://app.circleci.com/pipelines/github/Aggregate-Genius/periscope2?branch=master)
+[![Code Coverage](https://app.codecov.io/gh/Aggregate-Genius/periscope2/branch/master/graph/badge.svg)](https://app.codecov.io/gh/Aggregate-Genius/periscope2)
 <!-- badges: end -->
-  
-***periscope2*** is an extension of [periscope](https://cran.rstudio.com/web/packages/periscope/index.html) package. It provides a predefined but flexible template for new Shiny applications with a default dashboard layout, three locations for user alerts, a nice busy indicator and logging features. 
 
-One of the most important features of the shiny applications created with this framework is the separation by file of functionality that exists in one of the three shiny scopes: global, server-global, and server-local. The framework forces application developers to consciously consider scoping in Shiny applications by making scoping distinctions very clear without interfering with normal application development. Scoping consideration is important for performance and scaling, which is critical when working with large datasets and/or across many users.  In addition to creating a consistent UI experience this framework reduces development time for new applications by removing some of the boilerplate aspects of new applications such as alerting, logging, etc.
+### Overview
+
+***periscope2*** is a scalable and UI-standardized 'shiny' framework including a variety of developer convenience functions with the goal of both streamlining robust application development and assisting in creating a consistent user experience regardless of application or developer.
+
+
+***periscope2*** is full of developer-friendly features such as:
+
+* A predefined but flexible templates for new Shiny applications with a default [bs4Dash](https://bs4dash.rinterface.com/) layout
+* Separation by file of functionality that exists in one of the three shiny scopes: global, server-global, and server-local
+* Generated applications are organized in an easy to follow and maintain folder structure based on files functionality
+* Off shelf and ready to be used modules ('Table Downloader', 'Plot Downloader', 'File Downloader' and 'Reset Application'
```

Comment:
"Off shelf" should be "Off-the-shelf" (with dashes)

Reply:
done